Wikipedia/Localizations/ast.lproj/Localizable.strings

```diff
@@ -324,8 +324,7 @@
 "reading-list-api-error-list-limit" = "Esta llista nun ta sincronizada porque aportasti a la llende del númberu de llistes sincronizaes.";
 "reading-list-deleted-accessibility-notification" = "Desaniciada la llista de llectura";
 "reading-list-do-not-keep-button-title" = "Non, desaniciar los artículos del preséu";
-// Fuzzy
```

Comment:
One trick I use in cases where develop has moved ahead since I started a branch but before I've started to split things into individual commits is rebasing on develop before using the diff viewer to split into commits - that way my diff only reflects my changes.